Fuchas/Kernel/Startup/01_luaapi.lua

```diff
@@ -121,12 +58,11 @@ function string.trim(s)
 end
 
 function string.split(str, sep)
-	if sep == nil then
-		sep = "%s"
-	end
-	local t={}
+	sep = sep or "%s"
+	local t,n={},0
 	for part in string.gmatch(str, "([^"..sep.."]+)") do
-		table.insert(t, part)
+		t[n+1]=part
```

Comment:
why would that be better than `table.insert(t, part)` ?

Reply:
it's like 2-5x as fast

Reply:
oh ok

Reply:
```ocawesome101@manjaro-pbp: Fuchas$ time lua -e 'local t,n = {},0; for i=1, 2^27, 1 do n=n+1 t[n]=1 end'
real    0m10.623s
user    0m8.673s
sys     0m1.924s
ocawesome101@manjaro-pbp: Fuchas$ time lua -e 'local t = {}; for i=1, 2^27, 1 do table.insert(t,1) end'
real    1m37.906s
user    1m35.729s
sys     0m1.962s```